### DISCUSSION TITLE
如何使用串口通信？How to use serial communication?

### DISCUSSION THREAD

**Discussion by fxyc87** [vlang/v#12547] [General]
How to use serial communication?
help!!!

**Comment by fxyc87** [comment]
On Windows

**Comment by fxyc87** [comment]
已实现，
`
import io
import os

fn main(){
	print("123")
	mut com1:= os.create("COM1") or{
		print("打开COM1错误")
		return
	}
	com1.writeln("hello") or{
		print("写入COM1错误")
		return
	}
	
}
`

**Comment by fxyc87** [comment]
But I don't know how to set the baud rate？

**Comment by ghost** [comment]
you can join here for help
https://discord.com/channels/592103645835821068/592294828432424960
